packages/gatsby-theme-newrelic/src/components/GlobalStyles/GlobalStyles.js

```diff
@@ -125,9 +125,16 @@ const GlobalStyles = ({ layout }) => (
       }
 
       ul > li::marker {
-        color: var(--system-text-secondary-inverted-light);
+        color: var(--system-text-bullet-points);
         .dark-mode & {
-          color: var(--system-background-selected-low-contrast-dark);
+          color: var(--system-text-bullet-points-dark);
+        }
+      }
```

Comment:
no need to specify dark mode here, we can just use `--primary-bullet-points-color` since it has both a light and dark mode value in themes.js

Reply:
It was already like this before, I just changed the colors. Should I change it to `--primary-bullet-points-color` then?

Reply:
either way! themes.js lets us use a single variable that has both a light mode and dark mode value. we didn't use a theme var for bullet points before so that's why this code had to specify dark mode